src/cases/scorpio_wrapper.cpp

```diff
@@ -176,7 +187,8 @@ int scorpio_define_var(e3sm_io_config &cfg,
 #endif
 }
 
-int scorpio_write_var(e3sm_io_driver &driver,
+int scorpio_write_var(e3sm_io_config &cfg,
+                      e3sm_io_driver &driver,
```

Comment:
A better approach is to change cfg from a protected to a public member of
e3sm_io_driver in file e3sm_io_driver.hpp, so, we can simply use driver.cfg->rank
without changing the syntax of scorpio_write_var().

Reply:
OK, maybe you can make this change (protected to public) in the master branch before I will rebase this PR? Thanks.

Reply:
It is now in
https://github.com/Parallel-NetCDF/E3SM-IO/commit/dec3c4fb42bc79d656a83a0505bb7f7d5a8f2dde